examples/testing/keystone.ts

```diff
@@ -1,28 +1,28 @@
 import { config } from '@keystone-6/core';
 import { statelessSessions } from '@keystone-6/core/session';
 import { createAuth } from '@keystone-6/auth';
-import { lists } from './schema';
+import { models } from './schema';
 
 // createAuth configures signin functionality based on the config below. Note this only implements
-// authentication, i.e signing in as an item using identity and secret fields in a list. Session
+// authentication, i.e signing in as an item using identity and secret fields in a model. Session
```

Comment:
Maybe list here?? `withauth` won't work with a singleton... 
```suggestion
// authentication, i.e signing in as an item using identity and secret fields in a list. Session
```

Reply:
So, FUN thing - you can only auth with lists, so this is always a list, not a generic model